SPECS/openssl/openssl.spec

```diff
@@ -53,6 +53,7 @@ Patch29:        CVE-2023-0286.patch
 Patch30:        CVE-2022-4304.patch
 Patch31:        CVE-2022-4450.patch
 Patch32:        CVE-2023-0215.patch
+Patch33:        CVE-2023-0464.patch
```

Comment:
Thanks for validating AMD64! Have you also checked the package builds and tests succeed for ARM64?

Reply:
Nice catch. Running that right now and adding to the PR once it finishes.